charts/timed/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.3.11
-appVersion: v0.13.2
+version: 0.3.12
+appVersion: v1.0-rc1
```

Comment:
I guess this now needs a rc2 release after merging https://github.com/adfinis-sygroup/timed-backend/pull/586?

Reply:
yep, good point